### PR TITLE
Changed case of spelling

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

**Review by ArtificialOwl — APPROVED** [review]

**Comment by ArtificialOwl** [comment]
is it really needed to switch `\'` to `\"` in the vue files ? it seems to f*ck eslint

**Comment by rakekniven** [comment]
Yes, all messages in the entire NC project are aligned to use " instead of ' .

**Comment by ArtificialOwl** [comment]
@artonge any idea why this break eslint ?

**Review by Valdnet — CHANGES_REQUESTED** [review]

File: appinfo/info.xml [hunk 1]
```diff
@@ -9,13 +9,13 @@
 		 The Backup App creates and stores backup images of your Nextcloud:
 
 	  - Backup the instance, its apps, your data and your database,
```
Comment (Valdnet) [inline appinfo/info.xml:11]:
```suggestion
      - Backup the instance, its apps, your data and your database,
```

**Review by Valdnet — CHANGES_REQUESTED** [review]

**Review by artonge — APPROVED** [review]

**Comment by ArtificialOwl** [comment]
@rakekniven do you agreee with @artonge's changes ?
https://github.com/nextcloud/backup/pull/75/commits/4a36e5f5d89944441a64526c49ec782b192edc38

**Comment by rakekniven** [comment]
Yes, that is fine with me.

**Pull request merged**